handbook/people-ops/compensation/equity-faq.md

```diff
@@ -0,0 +1,69 @@
+# Sourcegraph options and equity compensation FAQ
+
+## Sourcegraph options
+
+Sourcegraph currently only provides equity to employees in the form of options.
```

Comment:
Since we differentiate between employees and contractors below, and since both receive options, I think it is clearer to use "teammates" here.

```suggestion
Sourcegraph currently only provides equity to teammates in the form of options.
```

Reply:
Oops thank you for catching this! Unlike most pages, this one actually intentionally uses a mix of those terms, but this should be teammates :)